routes/permafrost.py

```diff
@@ -0,0 +1,152 @@
+    for lyr in wms_targets:
+        urls.append(base_wms_url.format(lyr))
+    urls.append(
+        base_wfs_url.format(
+            "jorgenson_2008_pf_extent_ground_ice_volume", "GROUNDICEV,PERMAFROST"
+        )
+    )
+    urls.append(base_wfs_url.format("obu_pf_extent", "PFEXTENT"))
+
+    async with ClientSession() as session:
+        tasks = [fetch_layer_data(url, session) for url in urls]
+        results = await asyncio.gather(*tasks)
+    return results
+
+
+def package_gipl_magt(gipl_magt_resp):
+    """Package GIPL MAGT data in dict"""
+    gipl_magt_pkg = {}
+
+    for ix, i in enumerate(wms_targets[0:6]):
+        if gipl_magt_resp[ix]["features"] == []:
+            gipl_magt_resp["GIPL MAGT"] = "No data at this location."
+        else:
+            depth = i.split('_')[1] + '_'
+            yr = i.split('_')[-2] + '_'
+            key_str = 'GIPL_' + yr + depth + 'MAGT'
+            gipl_magt_pkg[key_str] = round(
+                gipl_magt_resp[ix]["features"][0]["properties"]["GRAY_INDEX"], 3
+            )
+    return gipl_magt_pkg
+
+
+def package_gipl_alt(gipl_alt_resp):
+    """Package GIPL ALT data in dict"""
+    gipl_alt_pkg = {}
+
+    for ix, i in enumerate(wms_targets[6:8]):
+        if gipl_alt_resp[ix]["features"] == []:
+            gipl_alt_resp["GIPL ALT"] = "No data at this location."
+        else:
+            yr = i.split('_')[-2] + '_'
+            key_str = 'GIPL_' + yr + 'ALT'
+            gipl_alt_pkg[key_str] = round(
+                gipl_alt_resp[ix]["features"][0]["properties"]["GRAY_INDEX"], 3
+            )
+    return gipl_alt_pkg
+
+
+def package_obu_magt(obu_magt_resp):
+    """Package Obu MAGT data in dict"""
+    obu_magt_pkg = {}
+
+    if obu_magt_resp["features"] == []:
+        obu_magt_resp["Obu MAGT"] = "No data at this location."
+    else:
+        key_str = 'Obu 2000-2016 MAGT (Top of Permafrost)'
+        obu_magt_pkg[key_str] = round(
+            obu_magt_resp["features"][0]["properties"]["GRAY_INDEX"], 3
+        )
+    return obu_magt_pkg
+
+
+def package_jorgenson(jorgenson_resp):
+    """Package Jorgenson data in dict"""
+    jorgenson_pkg = {}
+    if jorgenson_resp["features"] == []:
+        jorgenson_pkg["Jorgenson data"] = "No data at this location."
+    else:
+        jorgenson_pkg["Ground Ice Volume"] = jorgenson_resp["features"][0][
+            "properties"
+        ]["GROUNDICEV"]
+        jorgenson_pkg["Permafrost Extent"] = jorgenson_resp["features"][0][
+            "properties"
+        ]["PERMAFROST"]
+    return jorgenson_pkg
+
+
+def package_obu_vector(obu_vector_resp):
+    """Package obu_vector data in dict"""
+    obu_vector_pkg = {}
+    if obu_vector_resp["features"] == []:
+        obu_vector_pkg["Obu vector data"] = "No data at this location."
+    else:
+        obu_vector_pkg["Permafrost Extent"] = obu_vector_resp["features"][0][
+            "properties"
+        ]["PFEXTENT"]
+    return obu_vector_pkg
+
+
+@routes.route("/🧊/<lat>/<lon>")
```

Comment:
As much as it pains me to be serious, we probably should make this one use `permafrost` as an endpoint.

Reply:
We also need to add an Explainer / About template page for this.